go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/persistenceOne/persistenceBridge
 
-go 1.16
+go 1.17
```

Comment:
version should be same as version used by cosmos

Reply:
Why would we need one? In case of any changes in golang we would get a compile-time error. Also, golang has a guarantee of 100% backward capability.